src/plugins/telemetryTable/pluginSpec.js

```diff
@@ -49,7 +49,7 @@ describe("the plugin", () => {
     let element;
     let child;
 
-    beforeEach((done) => {
+    beforeAll((done) => {
```

Comment:
See previous comments. I really think we should be doing any state setup and teardown in `beforeEach` and `afterEach` for predictability.